src/BizHawk.Client.EmuHawk/tools/Lua/LuaConsole.cs

```diff
@@ -687,13 +687,9 @@ private void SaveSessionAs()
 			var file = GetSaveFileFromUser();
 			if (file != null)
 			{
-				var path = Config.PathEntries
-					.AbsolutePathFor(file.FullName, "")
-					.MakeRelativeTo(Path.GetDirectoryName(file.FullName));
-				LuaImp.ScriptList.Save(path);
-
-				Config.RecentLuaSession.Add(file.FullName); // TODO: should path be used here?
-				OutputMessages.Text = $"{Path.GetFileName(LuaImp.ScriptList.Filename)} saved.";
+				LuaImp.ScriptList.Save(file.FullName);
+				Config.RecentLuaSession.Add(file.FullName);
+				OutputMessages.Text = $"{file.Name} saved.";

```

Comment:
Do you believe https://github.com/TASEmulators/BizHawk/commit/99dc0e03df4c4cd20420507a06dd9987cbdf7140 broke this? I'm guessing you can reproduce this easily, so please check 2.4.2.

Reply:
Yes, what happened here is that the original code was supposed to make the script paths inside the .luases relative to the path of the .luases
Then 99dc0e03df4c4cd20420507a06dd9987cbdf7140 accidentally made *the path of the .luases* relative to itself, so they always end up as ".\foo.luases" and are saved in the current directory

Reply:
So this restores the old behaviour?